backend/src/services/secret-sharing/secret-sharing-service.ts

```diff
@@ -118,7 +137,10 @@ export const secretSharingServiceFactory = ({
       }
       await secretSharingDAL.updateById(sharedSecretId, { $decr: { expiresAfterViews: 1 } });
     }
-    return sharedSecret;
+    if (sharedSecret.accessType === SecretSharingAccessType.Organization) {
```

Comment:
I think we should still check to make sure the org of the logged in user match up? `sharedSecret.accessType === SecretSharingAccessType.Organization && orgId === sharedSecret.orgId`

Reply:
Redundancy added ; )